app/src/sharedTest/java/org/oppia/android/app/topic/lessons/TopicLessonsFragmentTest.kt

```diff
@@ -687,12 +689,12 @@ class TopicLessonsFragmentTest {
         explorationId = FRACTIONS_EXPLORATION_ID_0
         storyId = FRACTIONS_STORY_ID_0
         topicId = FRACTIONS_TOPIC_ID
-        profileId = ProfileId.newBuilder().apply { internalId = internalProfileId }.build()
+        profileId = this.profileId
         isCheckpointingEnabled = false
         parentScreen = ExplorationActivityParams.ParentScreen.TOPIC_SCREEN_LESSONS_TAB
       }.build()
       intended(
-        allOf(
+        anyOf(
```

Comment:
Hi Ben using allOf checks that particular key but since in this refactor profileId is wrapped into its own key 'allOf' will keep failing updating check to 'anyOf' will make it pass since we have 2 keys holding proto data. The check for 'profileId' is being done elsewhere on other tests.